app/views/datasets/_datafile_table.html.erb

```diff
@@ -6,39 +6,45 @@
     <th>Data preview</th>
   </tr>
   <tbody>
-    <% datafiles.each do |datafile| %>
-      <tr>
-        <td class="title small">
-          <%= link_to (datafile.name ? datafile.name : 'Data Link'),
-              datafile.url,
-              :data => {:'ga-event' => "datafile-download"}
-          %>
-        </td>
-        <% if datafile.format.blank? %>
-          <td class="dgu-unavailable">N/A</td>
-        <% else %>
-          <td><%= datafile.format.upcase %>
+  <% datafiles.each do |datafile| %>
+    <tr class="<%= show_more?(datafiles, datafile) %> dgu-datafile">
+      <td class="title small">
+        <%= link_to (datafile.name ? datafile.name : 'Data Link'),
+                    datafile.url,
+                    :data => {:'ga-event' => "datafile-download"}
+        %>
+      </td>
+      <% if datafile.format.blank? %>
+        <td class="dgu-unavailable">N/A</td>
+      <% else %>
+        <td><%= datafile.format.upcase %>
           <% if datafile.size %>
             (<%= datafile.size %> kB)
           <% end %>
-          </td>
-        <% end %>
+        </td>
+      <% end %>
 
-        <% if datafile.updated_at.present? %>
-          <td><%= format(datafile.updated_at) %></td>
+      <% if datafile.updated_at.present? %>
+        <td><%= format(datafile.updated_at) %></td>
+      <% else %>
+        <td class="dgu-unavailable">Not available</td>
+      <% end %>
+      <td class="datafiles__preview">
+        <% if datafile.html? %>
+          <%= link_to 'Go to site', datafile.url %>
+        <% elsif datafile.csv? %>
+          <span class="dgu-unavailable">Coming soon</span>
         <% else %>
-          <td class="dgu-unavailable">Not available</td>
+          <span class="dgu-unavailable">Not available</span>
         <% end %>
-          <td class="datafiles__preview">
-            <% if datafile.html? %>
-              <%= link_to 'Go to site', datafile.url %>
-            <% elsif datafile.csv? %>
-              <span class="dgu-unavailable">Coming soon</span>
-            <% else %>
-              <span class="dgu-unavailable">Not available</span>
-            <% end %>
-          </td>
-        </tr>
-      <% end %>
+      </td>
+    </tr>
+  <% end %>
   </tbody>
 </table>
+
+<% if datafiles.length > 5 %>
+  <p class="show-toggle">
```

Comment:
this shoud be a button, for accessibility reason. Hence you can reuse the button.secondary class from govuk-overrides.scss

Reply:
Cool - done